packages/config/config/devices/0x0208/hkzw_ms01.json

```diff
@@ -1,7 +1,7 @@
 {
 	"manufacturer": "HANK Electronics Ltd.",
 	"manufacturerId": "0x0208",
-	"label": "HKZW_MS01",
+	"label": "HKZW MS01",
```

Comment:
In the other files you used a dash, here it is a space. Intended?

Reply:
You're correct. This should have a dash for consistency! I will fix  this.